lib/torrent.js

```diff
@@ -476,23 +476,18 @@ class Torrent extends EventEmitter {
 
     this._rarityMap = new RarityMap(this)
 
+    this.files = this.files.map(file => new File(this, file))
+
     this.store = new ImmediateChunkStore(
       this._preloadedStore || new this._store(this.pieceLength, {
-        torrent: {
-          infoHash: this.infoHash
-        },
-        files: this.files.map(file => ({
-          path: path.join(this.path, file.path),
```

Comment:
isn't this lost?

Reply:
the file object includes it's [relative path](https://github.com/webtorrent/webtorrent/blob/master/lib/file.js#L20), torrent object includes its infohash too